### PR TITLE
Reuse workspace activity managing components from Che

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Use idling tracking from Che (https://github.com/eclipse/che/pull/5512)

### What issues does this PR fix or reference?
https://github.com/eclipse/che/pull/5386

#### Changelog
Activity tracking components are now re-used from Che. Configuration properties remain the same for Codenvy.

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

**Comment by slemeur** [comment]
change log , docs are missing

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/codenvy-api-configuration.properties [hunk 1]
```diff
@@ -43,8 +43,6 @@ password.recovery.expiration_timeout_hours=1
 # path to templates' description folder
 che.template.storage=${catalina.base}/templates
 
-stop.workspace.scheduler.period=60
```
Comment (garagatyi) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/codenvy-api-configuration.properties]: Looks like this property is missing now

Comment (mkuznyetsov) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/codenvy-api-configuration.properties]: added property back

**Comment by mkuznyetsov** [comment]
@slemeur Added Changelog. There are no changes needed for docs, as this feature will still works in the same way.

**Pull request merged**